src/components/chip/angularjs/chip_directive.js

```diff
@@ -121,6 +122,24 @@ function ChipDirective() {
         if (transclude.isSlotFilled('label')) {
             ctrl.hasLabel = true;
         }
+
+        const defaultProps = {
+            color: 'dark',
```

Comment:
Just to confirm, `dark` is a color or a theme? Looking at the template, it looks like you wanted to make the default theme `dark`?

Reply:
Don't know about angular version, @matmkian ?

Reply:
No more default value for theme prop
Add color prop with dark default value -> dark is a theme but also a color :-)

Reply:
:wtfception: